src/Components/Playlist/Playlist.js

```diff
@@ -0,0 +1,38 @@
+import React, { useContext } from 'react';
+import TrackList from '../TrackList/TrackList';
+import { spotifySave } from '../../util/Spotify';
+import { PlaylistContext } from '../Context/PlaylistContext';
+import './Playlist.css';
+
+export default function Playlist() {
+
+    const infoContext = useContext(PlaylistContext);
+
+    function updatePlaylistName(name) {
+        infoContext.setPlaylistName(name);
```

Comment:
Prefer passing Context into functions and having functions outside of the component. Easier to write tests